src/pages/Projects.jsx

```diff
@@ -86,8 +86,8 @@ function Projects() {
       </div>
       <LetterButtonsContainer onSetLetter={(letter) => setLetter(letter)} />
       <div className="accordion-wrap projects">
-        {projectList.map((project, index) => (
-          <Project id={project.id} key={index} />
+        {projectList.map((project) => (
+          <Project id={project.id} key={project.id} />
```

Comment:
This code was the issue for the bug. React uses key prop to keep track of whether the elements in the list are same or not so he doesn't have to rerender them. It is a bad practice to use index as a key in lists that can be reordered by deletion, drag and so on. 
Take a look at this video from this timestamp and watch until minute 24:
https://youtu.be/i793Qm6kv3U?t=771

Reply:
You always want to put unique values such as project.id in the key prop.